### ISSUE TITLE
useCallback received a final argument...

### ISSUE THREAD

**Issue by sibuser** [tannerlinsley/react-charts#41]
I get this error when include a Chart component into my project.
```
useCallback received a final argument that is not an array (instead, received `function`). When specified, the final argument must be an array.
```

```
function MyChart(){
  const data = React.useMemo(
    () => [
      {
        label: "Series 1",
        data: [[0, 1], [1, 2], [2, 4], [3, 2], [4, 7]]
      },
      {
        label: "Series 2",
        data: [[0, 3], [1, 1], [2, 5], [3, 6], [4, 4]]
      }
    ],
    []
  );

  const series = React.useMemo(
    () => ({
      showPoints: false
    }),
    []
  );

  const axes = React.useMemo(
    () => [
      { primary: true, type: "linear", position: "bottom" },
      { type: "linear", position: "left" }
    ],
    []
  );
return (
      <div
        style={{
          width: "400px",
          height: "300px"
        }}
      >
        <Chart data={data} series={series} axes={axes} tooltip />
      </div>
)
}
```

**Comment by sibuser** [comment]
I've managed to reproduce it with these React version

https://codesandbox.io/s/happy-vaughan-wf5pj

**Comment by keeganstothert** [comment]
i have the exact same issue

**Comment by luzma87** [comment]
having the same issue.. react 16.11.0 and react-charts 2.0.0-beta.4 :/

**Comment by sbuddala-fraedom** [comment]
I am having the same issue. "react": "16.12.0",  "react-charts": "2.0.0-beta.4"

**Comment by Sinamics** [comment]
having the same issue. "react": "^16.12.0",  "react-charts": "^2.0.0-beta.4",

**Comment by chfritz** [comment]
The error stack shows it's an issue in here:
```
  getSeriesID = React.useCallback(Utils.normalizeGetter(getSeriesID), getSeriesID);
```
indeed `getSeriesID` seemed to have been a function in 2.0.0-beta.5, **but** this now seems to be fixed in 2.0.0-beta.6.

**Comment by tannerlinsley** [comment]
Beta doesn't even use this API/function any more, so I'm going to close this.
